hack/run-e2e.sh

```diff
@@ -48,25 +48,26 @@ function run_test {
     --output-dir ${ARTIFACTS}/results/ \
     --focus "${FOCUS}" \
     --skip "${SKIP}" \
-    $EXTRA_ARGS| tee /tmp/test.log
-  
+    --namespace "${NAMESPACE}" \
```

Comment:
Will it be an issue if namespace is set to an empty string? I don't see any validation to set it to the default. Does Viper take care of this somehow?

Reply:
I followed the same logic we already have for the FOCUS/SKIP/DRY_RUN variables further down. Besides the kind stuff, I can run the `run-e2e.sh` locally just fine.

Reply:
Sounds good, let's give it a shot.